include/operations/blas2_trees.h

```diff
@@ -347,6 +347,44 @@ make_tbmv(lhs_t &lhs_, matrix_t &matrix_, typename vector_t::index_t k_,
               is_unit>(lhs_, matrix_, k_, vector_);
 }
 
+/**
+ * @struct Trsv
+ * @brief Tree node representing a triangular matrix_ vector_
+ * multiplication.
+ */
+template <typename lhs_t, typename matrix_t, typename vector_t,
+          uint32_t local_range, bool is_upper, bool is_transposed, bool is_unit>
```

Comment:
Why do we need this `uint32_t` type here?

Reply:
We need it for passing the block size as a parameter, such as for `gemv`.